.github/workflows/check-link.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   lychee:
     runs-on: ubuntu-24.04
     env:
-      CLI_VERSION: '0.18.1' # selfup {"extract":"\\d\\.\\d+\\.\\d+","replacer":["lychee", "--version"], "nth": 2}
+      CLI_VERSION: '0.18.0' # selfup {"extract":"\\d\\.\\d+\\.\\d+","replacer":["lychee", "--version"], "nth": 2}
```

Comment:
Downgrading lychee making CI error for https://www.kioxia.com/ja-jp/personal/software/ssd-utility.html 🤔

Reply:
Fixed in https://github.com/kachick/dotfiles/pull/1086/commits/104c871b7bc96626e708b7c44382bb40c94cf98b